ComSemApp/templates/ComSemApp/teacher/attempt_request_info.html

```diff
@@ -0,0 +1,105 @@
+{% load static %}
+<div class="row h-100">
+    <div class="col-12 h-100">
+        <div class="card-box">
+            <div class="card">
+                <div class='card-body'>
+                    <!--card for displaying results-->
+                    <b>Student</b><br>
+                    {{attempt.student}} <br>
+                    <b>Course</b><br>
+                    {{ course.session }}<br>
+                    <b>Correct Expression</b><br> 
+                    {{attempt.expression.reformulation_text}} <br>
+                    <b>Problem Expression</b><br> 
+                    {{attempt.expression.expression}} <br>
+                    <b>Graded Expression</b> (hover for details about each error)<br> 
+                    <mark><span title="There could be errors shown here">{{attempt.transcription}}</span></mark><br>
+                    <b>Audio Recording</b><br>
+                    <audio controls title="Reformulation" style="width: 200px" class="previous_attempt_audio" src="{{ attempt.audio.url }}"></audio><br>
+                    <b>Fluency Score</b><br>
+                    {{attempt.wpm}}<br>
+                    <b>Correctness Score</b><br>
+                    {% if review.original_score%}
+                        {{ review.original_score}}
+                    {% else %}
+                        {{attempt.correct}}
+                    {% endif %}
+                    <br>
+                    <b>New Correctness Score <input type="number" min="0" max="100" style="width:85px;" id="myPercent" {% if review.original_score %}value="{{ attempt.correct }}"{% endif %}></b>
+                    <div id="myTextboxContainer"></div>
+                    <div style="position: relative;">
+                        <b>Comments</b><br>
+                        <textarea id="teacher_comments_box" style="width:100%;" rows="4">{{ review.comments }}</textarea>
+                    </div>
+                    <br>
+                    
+                    {% if review %}
+                        <button id="saveChanges">Save Changes</button>
+                    {% else %}
+                        <button id="submit">Submit</button>
+                    {% endif %}
+                </div>
+            </div>
+        </div>
+    </div>
+</div> 
+<script>
+let reviewer = "{{ user.id | safe }}";
+
+function isNumber(value) {
+    return !isNaN(parseFloat(value));
+  }
+
+function submitForm(url){
+    /*
+        takes current data and send it in a post request to the server (SpeakingPracticeAttemptCreateView)
+    */
+    var newScore = $('#myPercent').val();
+    if (newScore != ""){
+        if (!isNumber(newScore) || newScore < 0 || newScore > 100) {
```

Comment:
The isNumber shouldn't be needed. There is way to enforce numbers only in html forms: https://stackoverflow.com/questions/13952686/how-to-make-html-input-tag-only-accept-numerical-values